ios/web_view/internal/web_view_global_state_util_stub.mm

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Comment:
why does the filename have `_stub`?

Reply:
Just stubs out the `InitializeGlobalState` method so I added `_stub` but I can remove that

Reply:
fixed in https://github.com/brave/brave-core/pull/26957/commits/b21aacfbfbc91e4701cabf59cff6ca37c078f4e6